Automattic/wp-calypso
# Show Google Plus Deprication Message

#### Changes proposed in this Pull Request

* Disable adding new Google Plus connections by removing button
* Include a link (URL TBD) explaining why this functionality has been removed

#### Testing instructions

* Goto: http://calypso.localhost:3000/sharing/{DOMAINSLUG}
* Observe that you can now longer connect to Google Plus
* Sharing via existing Google Plus connections should still work

Non-connected account:
![screen shot 2018-12-13 at 1 53 29 pm](https://user-images.githubusercontent.com/6817400/49960646-a341e680-fede-11e8-9b5c-dd0202769588.png)

Connected Account:
![screen shot 2018-12-13 at 1 19 08 pm](https://user-images.githubusercontent.com/6817400/49958854-054c1d00-feda-11e8-8087-25fa60a40ce6.png)
